ads/google/test/test-doubleclick.js

```diff
@@ -0,0 +1,363 @@
+    it: '(multi-size) should render an ad without resizing',
+  });
+});
+
+describe('multi-size doubleclick ad request, below the fold', function() {
+  this.timeout(waitTime);
+  doubleClickTest({
+    html: createFixtureHtml({
+      width: 480,
+      height: 75,
+      multiSize: '320x50',
+    }, true),
+    multiSize: '320x50',
+    primarySize: {height: '75', width: '480'},
+    actualCreativeSize: {height: '50', width: '320'},
+    expectResize: true,
+    pageUrl: 'http://nonblocking.io/',
+    adIframeId: 'google_ads_iframe_/4119129/mobile_ad_banner_0',
+    it: '(multi-size) should render an ad with resizing',
+  });
+});
+
+describe(
+    'multi-size doubleclick ad request, secondary size larger than primary',
+    function() {
+      this.timeout(waitTime);
+      doubleClickTest({
+        html: createFixtureHtml({
+          width: 480,
+          height: 75,
+          multiSize: '620x50',
+        }, false),
+        multiSize: '620x50',
+        primarySize: {height: '75', width: '480'},
+        actualCreativeSize: {height: '50', width: '320'},
+        expectResize: false,
+        pageUrl: 'http://nonblocking.io/',
+        adIframeId: 'google_ads_iframe_/4119129/mobile_ad_banner_0',
+        it: '(multi-size) ad rendering should fail',
+        expectEmpty: true,
+      });
+    });
+
+describe('multi-size doubleclick ad request, ' +
+    'secondary size less than 2/3rds of primary',
+    function() {
+      this.timeout(waitTime);
+      doubleClickTest({
+        html: createFixtureHtml({
+          width: 900,
+          height: 100,
+          multiSize: '320x50',
+        }, false),
+        multiSize: '320x50',
+        primarySize: {height: '100', width: '900'},
+        actualCreativeSize: {height: '50', width: '320'},
+        expectResize: false,
+        pageUrl: 'http://nonblocking.io/',
+        adIframeId: 'google_ads_iframe_/4119129/mobile_ad_banner_0',
+        it: '(multi-size) ad rendering should fail',
+        expectEmpty: true,
+      });
+    });
+
+describe('multi-size doubleclick ad request, ' +
+    'secondary size less than 2/3rds of primary but validation to be ' +
+    'ignored, above fold',
+    function() {
+      this.timeout(waitTime);
+      doubleClickTest({
+        html: createFixtureHtml({
+          width: 900,
+          height: 100,
+          multiSize: '320x50',
+          multiSizeValidation: 'false',
+        }, false),
+        multiSize: '320x50',
+        primarySize: {height: '100', width: '900'},
+        actualCreativeSize: {height: '50', width: '320'},
+        expectResize: false,
+        pageUrl: 'http://nonblocking.io/',
+        adIframeId: 'google_ads_iframe_/4119129/mobile_ad_banner_0',
+        it: '(multi-size) ad should render',
+      });
+    });
+
+describe('multi-size doubleclick ad request, ' +
+    'secondary size less than 2/3rds of primary but validation to be ' +
+    'ignored, below fold',
+    function() {
```

Comment:
Not sure if I'm doing something wrong, but if I use arrow functions, I run into a scoping issue where  this.timeout is undefined.